version.txt

```diff
@@ -1 +1 @@
-1.28.0
+1.29.0
```

Comment:
Still need to remove this from this commit, this commit should only update the dependencies

Reply:
ack

Reply:
Updated the PR